test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -102,9 +104,27 @@ private void Context_Log(object sender, LoggingEventArgs e)
             //throw new NotImplementedException();
         }
 
-        [Fact]
-        public void AutoFitRegressionTest()
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void AutoFitRegressionTest(bool foreignCulture)
         {
+            var originalCulture = Thread.CurrentThread.CurrentCulture;
+            uint experimentTime = 0;
+
+            if (foreignCulture)
```

Comment:
`pl-PL` is not foreign for people in Poland; perhaps a more generic name?
```suggestion
            if (internationalizationTest)
```

Reply:
Alternatively, we could pass in which culture to test against? 

Then do a for loop over the specific cultures we want to test: `{ "en-US", "ar-SA" }`.

`ar-SA` is interesting as they have both a different representation of negative numbers `1-`, and a uses a comma as the decimal separator `0,23`. This test both long and float hyperparameters, whereas `pl-PL` only tests the floats.

Reply:
that's an interesting idea, I'll try it out

Reply:
Thanks.
#Resolved